[debug] [2017-09-20T09:35:35.859Z] ----------------------------------------------------------------------
[debug] [2017-09-20T09:35:35.862Z] Command:       E:\nodejs\node.exe C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login
[debug] [2017-09-20T09:35:35.863Z] CLI Version:   3.12.0
[debug] [2017-09-20T09:35:35.863Z] Platform:      win32
[debug] [2017-09-20T09:35:35.863Z] Node Version:  v6.10.0
[debug] [2017-09-20T09:35:35.864Z] Time:          Wed Sep 20 2017 17:35:35 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:35:35.864Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=907776254&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2017-09-20T09:36:14.273Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token { code: '4/lztq4XZrkO9ORWJ0MNYGte0QE1M8xW0n0VuMaREhXBg',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Wed Sep 20 2017 17:36:14 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:36:35.341Z] Token Fetch Error: Error
    at Error.FirebaseError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:32:23)
    at self.callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:310:9)
    at emitOne (events.js:96:13)
[debug] [2017-09-20T09:38:58.688Z] ----------------------------------------------------------------------
[debug] [2017-09-20T09:38:58.691Z] Command:       E:\nodejs\node.exe C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login --interactive
[debug] [2017-09-20T09:38:58.692Z] CLI Version:   3.12.0
[debug] [2017-09-20T09:38:58.692Z] Platform:      win32
[debug] [2017-09-20T09:38:58.692Z] Node Version:  v6.10.0
[debug] [2017-09-20T09:38:58.693Z] Time:          Wed Sep 20 2017 17:38:58 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:38:58.693Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=386141194&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2017-09-20T09:39:11.813Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token { code: '4/4gTdXCNZXhHeinUfFQ_MPP_QZ2uR7XF30BZ7ZrKITog',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Wed Sep 20 2017 17:39:11 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:39:32.849Z] Token Fetch Error: Error
    at Error.FirebaseError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:32:23)
    at self.callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:310:9)
    at emitOne (events.js:96:13)
[debug] [2017-09-20T09:41:15.156Z] ----------------------------------------------------------------------
[debug] [2017-09-20T09:41:15.159Z] Command:       E:\nodejs\node.exe C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login
[debug] [2017-09-20T09:41:15.160Z] CLI Version:   3.12.0
[debug] [2017-09-20T09:41:15.160Z] Platform:      win32
[debug] [2017-09-20T09:41:15.160Z] Node Version:  v6.10.0
[debug] [2017-09-20T09:41:15.161Z] Time:          Wed Sep 20 2017 17:41:15 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:41:15.161Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=845994731&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2017-09-20T09:42:40.148Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token { code: '4/rFBLB6CprguHXLBYl36gwrrWs8-6SOB3uAniS2G9_p8',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Wed Sep 20 2017 17:42:40 GMT+0800 (中国标准时间)
[debug] [2017-09-20T09:43:01.214Z] Token Fetch Error: Error
    at Error.FirebaseError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:32:23)
    at self.callback (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (C:\Users\kekehao\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:310:9)
    at emitOne (events.js:96:13)
